project/example_file.py

```diff
@@ -0,0 +1,13 @@
+from typing import List, Dict
+
+def some_func(arg_1: List, arg_2: Dict) -> None:
```

Comment:
I would sugges to do the following
```suggestion
def some_func(arg_1: List[str], arg_2: Dict) -> None:
```